packages/service-worker-mock/models/Request.js

```diff
@@ -34,7 +34,7 @@ class Request extends Body {
   }
 
   clone() {
-    if (this.bodyUsed) throwBodyUsed('json');
```

Comment:
Is this change related?

Reply:
No, it was something I saw while going through the code. So I made an additional commit to clean it up, cause I guess it has no effect?

Reply:
Looks like you're right! Good catch :)